### PR TITLE
Fix minor punctuation errors

### PR DESCRIPTION
**Pull request by msespos**
- [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md).

#### 1.Describe the changes made:

Fix three minor punctuation errors in the Rails Callbacks Article

#### 2. If this PR is related to an open issue please reference it with the `#` operator and the issue number below:

Not related to an open issue

### CODE REVIEW

File: rails_programming/advanced_forms_and_activerecord/active_record_callbacks.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ You can also use conditional logic options `:if` and `:unless` to try a method b
 
 ### Transaction Callbacks
 
-Sometimes your Rails app will need to interact with an external application (which is inherently imperfect) as a part of the save process.  Other times your save will involve juggling several balls at once and, if one fails, they all need to be rolled back.  Typically these cases will involve wrapping your database save operation in a "transaction", which means that either all the steps work or they all fail and are rolled back.
+Sometimes your Rails app will need to interact with an external application (which is inherently imperfect) as a part of the save process.  Other times your save will involve juggling several balls at once and, if one fails, they all need to be rolled back.  Typically these cases will involve wrapping your database save operation in a "transaction," which means that either all the steps work or they all fail and are rolled back.
```
Comment (KevinMulhern) [inline rails_programming/advanced_forms_and_activerecord/active_record_callbacks.md:72]: should the comma be within the quotes?

Comment (msespos) [reply rails_programming/advanced_forms_and_activerecord/active_record_callbacks.md:72]: As far as I understand it, commas do always go inside quotation marks. Here are two relevant references:
https://style.mla.org/punctuation-and-quotation-marks/
https://www.grammarly.com/blog/quotation-marks/
Let me know if you find anything different!

Comment (KevinMulhern) [reply rails_programming/advanced_forms_and_activerecord/active_record_callbacks.md:72]: ahh TIL, thanks for the explanation @msespos

**Review by KevinMulhern — APPROVED** [review]
LGTM 👍  Thanks for taking the time to do this @msespos

**Pull request merged**